ang/test/civicase/case/actions/services/webforms-case-action.service.spec.js

```diff
@@ -23,24 +25,102 @@
         });
       });
 
-      describe('when used inside case details page', function () {
+      describe('when webforms should not be visible in a separate list', function () {
         beforeEach(function () {
-          attributes.mode = 'case-details';
+          webformsList.isVisible = false;
         });
 
-        describe('when webforms are present', () => {
-          it('displays the action link', function () {
-            expect(WebformsCaseAction.isActionAllowed(webformAction, cases, attributes)).toBeTrue();
+        describe('webforms list in case actions dropdown', () => {
+          beforeEach(function () {
+            attributes.mode = 'case-details';
+          });
+
+          describe('when webforms are present', () => {
+            it('displays the action link', function () {
+              expect(WebformsCaseAction.isActionAllowed(webformAction, cases, attributes)).toBeTrue();
+            });
+          });
+
+          describe('when webforms are not present', () => {
+            beforeEach(function () {
+              webformAction.items = [];
+            });
+
+            it('hides the action link', function () {
+              expect(WebformsCaseAction.isActionAllowed(webformAction, cases, attributes)).toBeFalse();
+            });
           });
         });
 
-        describe('when webforms are not present', () => {
+        describe('webforms list in case summary header', () => {
           beforeEach(function () {
-            webformAction.items = [];
+            attributes.mode = 'case-details-header';
+          });
+
+          describe('when webforms are present', () => {
+            it('hides the action link', function () {
+              expect(WebformsCaseAction.isActionAllowed(webformAction, cases, attributes)).toBeFalse();
+            });
           });
 
-          it('displays the action link', function () {
-            expect(WebformsCaseAction.isActionAllowed(webformAction, cases, attributes)).toBeFalse();
+          describe('when webforms are not present', () => {
+            beforeEach(function () {
+              webformAction.items = [];
+            });
+
+            it('hides the action link', function () {
+              expect(WebformsCaseAction.isActionAllowed(webformAction, cases, attributes)).toBeFalse();
+            });
+          });
+        });
```

Comment:
are we missing the scenario to display the webform list in the case summary header? I only see scenarios for not showing it.

Reply:
"when webforms should not be visible in a separate list" "webforms list in case summary header" should not be visible.

Visible scenarios are present in "when webforms should not be visible in a separate list" "webforms list in case actions dropdown"